ParquetClassLibrary/Sandbox/MapRegion.cs

```diff
@@ -399,27 +397,28 @@ public ParquetStack GetAllParquetsAtPosition(Vector2Int in_position)
                                     GetCollectableAtPosition(in_position));
         }
 
-
         /// <summary>
         /// Gets all the parquets in the entire region.
         /// </summary>
         /// <returns>A collection of parquets.</returns>
         public IEnumerable<ParquetParent> GetAllParquets()
         {
+            // TODO: This is not optimal — it'd be nice to not generate these garbage objects full of duplicates
```

Comment:
This likely isn't generating any garbage, since it's just a list of pointers to objects. (if my CS knowledge is holding up)

Had offline chat about this; seems pretty fine.

Reply:
(Paige adds: Doublecheck that this list isn't being filled with uneeded empties btw)

Reply:
Thanks so much for the discussion surrounding my concerns here!